lms/djangoapps/courseware/toggles.py

```diff
@@ -38,3 +38,26 @@
 COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW = CourseWaffleFlag(
     WAFFLE_FLAG_NAMESPACE, 'microfrontend_course_team_preview', __name__
 )
+
+# Waffle flag to enable the course exit page in the learning MFE.
+#
+# .. toggle_name: courseware.microfrontend_course_exit_page
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Supports staged rollout of the new micro-frontend-based implementation of the course exit page.
+# .. toggle_category: micro-frontend
+# .. toggle_use_cases: incremental_release, open_edx
+# .. toggle_creation_date: UPDATE ME BEFORE MERGE
```

Comment:
reminder

Reply:
I'll just set to tomorrow. Thanks for the reminder